docs/recipes/Code-Splitting-Using-Holocron.md

```diff
@@ -1,7 +1,240 @@
+[holocron-module-route-readme]: https://github.com/americanexpress/holocron/tree/master/packages/holocron-module-route/README.md
+[one-app-bundler-readme]: https://github.com/americanexpress/one-app-cli/tree/master/packages/one-app-bundler/README.md
+[one-app-router-readme]: https://github.com/americanexpress/one-app-router/README.md
+[frank-lloyd-root]: ../../prod-sample/sample-modules/frank-lloyd-root/0.0.0/README.md
+[franks-burgers]: ../../prod-sample/sample-modules/franks-burgers/0.0.0/README.md
+
 [👈 Return to Overview](./Recipes.md)
 
 # Code Splitting Using Holocron
 
-Recipe is forthcoming.
+For One App, Holocron modules are a formal way of code-splitting for a modern,
+micro front-end in the One App ecosystem. Utilizing the module map as a system
+which promotes synergy and asynchrony across teams and collaborators on
+a single application.
+
+With Holocron each individual module is by definition, a code split bundled
+component with language packs that can be combined and composed based on our
+needs. Each module is versioned, configurable, routable and can be further
+split into smaller chunks using dynamic importing.
+
+Like `@loadable/component` and `react-loadable`, `holocron` is a complete code-splitting
+solution that works in both client and server side, however there is no need to scaffold
+a build infrastructure to support code-splitting when we use `one-app-bundler`.
+
+**[Read the `holocron` documentation.][holocron-readme]**
+
+**[Read the `one-app-bundler` documentation.][one-app-bundler-readme]**
+
+## Route based code splitting
+
+> To view a root Holocron module with a childConfig, [check out the sample module "Frank Lloyd Root"][frank-lloyd-root]
+
+A Holocron module can be route-driven using `ModuleRoute` and we can use this component
+to shape our routing for One App. Since each Holocron module is an independent bundle,
+our `childRoutes` is a map of code-split modules that is driven by the url. `ModuleRoute`
+extends `Route` from `one-app-router` and can render the module(s) by its route per
+`history`.
+
+**[Read mode about `holocron-module-route`][holocron-module-route-readme]**
+
+**[Read mode about `one-app-router`][holocron-module-route-readme]**
+
+Our code split Holocron modules are now loaded based on routing to the application.
+
+`src/childRoutes.jsx`
+```jsx
+import React from 'react';
+import { ModuleRoute } from 'holocron-module-route';
+
+export const childRoutes = () => (
+  <ModuleRoute
+    path="home"
+    moduleName="holocron-reactor"
+    onEnter={(nextState, replace, callback) => {
+      // ... do something on enter
+      callback();
+    }}
+  />
+);
+```
+
+or can be an elaborate route configuration:
+
+```jsx
+import React from 'react';
+import { RenderModule } from 'holocron';
+import { ModuleRoute } from 'holocron-module-route';
+
+export const childRoutes = () => (
+  <ModuleRoute moduleName="holocron-reactor">
```

Comment:
Shouldn't we use an example that is common to technology rather than scifi? I love scifi but what about demo-ing something like a social media website and how those pieces could fit into Modules?

Reply:
That's a great idea, I think that would be more helpful. Will update

Reply:
updated in cb5e958